packages/editor-ui/src/__tests__/render.ts

```diff
@@ -32,7 +32,6 @@ const defaultOptions = {
 			'vue-json-pretty': vueJsonPretty,
 		},
 		plugins: [
-			I18nPlugin,
 			i18nInstance,
```

Comment:
is this also still needed?

Reply:
We use that for the i18n composable. 

useI18n composable [here](https://github.com/n8n-io/n8n/blob/fd3254d5874a03b57421246b77a519787536a93e/packages/editor-ui/src/composables/useI18n.ts#L3
), return i18n, which is an instance of `I18nClass` `[here](https://github.com/n8n-io/n8n/blob/fd3254d5874a03b57421246b77a519787536a93e/packages/editor-ui/src/plugins/i18n/index.ts#L27)` and the `I18nClass` uses `createI18n` from `vue-i18n`

Reply:
But anyways that should be in another PR